deploy/dashboards/metrics/ibm-websphere-liberty-grafana-dashboard-metrics-2.0.json

```diff
@@ -1917,8 +2010,8 @@
       "type": "row"
     }
   ],
-  "refresh": "10s",
-  "schemaVersion": 16,
+  "refresh": false,
```

Comment:
should leave refresh time as 10s

Reply:
I've changed the refresh time back to "10s"